api/src/queries.ts

```diff
@@ -78,6 +85,30 @@ export class Connection {
     });
   }
 
+  async getAllPapers() {
+    const response = await this._knex.raw<{ rows: PaperWithEntityCounts[] }>(`
+      SELECT paper.*,
+             (
+                SELECT COUNT(*)
+                  FROM citation
+                 WHERE citation.paper_id = paper.s2_id
+             ) AS citations,
+             (
+                SELECT COUNT(*)
+                  FROM symbol
+                 WHERE symbol.paper_id = paper.s2_id
+             ) AS symbols
+        FROM paper
+    ORDER BY symbols DESC, citations DESC
+    `);
```

Comment:
I couldn't get `Knex` and a subquery to play nice, so I had to just write good ole' SQL.  That said, I like writing good ole' SQL :).